### PR TITLE
If user go to refer

### PR DESCRIPTION
**Pull request by RanSolo**
Where if there was a duplicate email account entered it used to take you to the homepage, now it goes to the existing users refer-to-friend page 

This behavior is working properly in this pr. If user exists then take them to their refer a friend page...

It is a much cleaner user experience and takes away ambiguity.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -2,6 +2,10 @@ class UsersController < ApplicationController
   before_filter :skip_first_page, only: :new
 
   def new
+    if !@user.nil?
+      cookies[:h_email] = { value: @user.email }
+      format.html { redirect_to '/refer-a-friend' }
+    end
     @bodyId = 'home'
```
Comment (RanSolo) [inline app/controllers/users_controller.rb]: This is the only significant change

**Comment by RanSolo** [comment]
Here I'm using a duplicate email, notice I'm saving the sign from outside the basement.  When clicked through the picture is the logo that I uploaded when using the email the first time. [https://gyazo.com/1615f81ef3a012cd26b1428682124fac](https://gyazo.com/1615f81ef3a012cd26b1428682124fac)

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@ class UsersController < ApplicationController
   before_filter :skip_first_page, only: :new
 
   def new
+    if !@user.nil?
+      format.html { redirect_to '/refer-a-friend' }
+    end
```
Comment (jchapin) [inline app/controllers/users_controller.rb]: Unless there is no user defined, redirect immediately to the refer-a-friend page.

**Comment by jchapin** [comment]
That should keep people from attempting to submit a duplicate. :+1:

**Pull request merged**